Source Code/Ticket Management System/Input_manip.c

```diff
@@ -0,0 +1,22 @@
+#include  "Input_manip.h"
```

Comment:
Doesn't hurt to keep file names lower case for consistency.

Reply:
I realized it's an error, but the studio didn't let me change it....

Reply:
Fixed*